bedwars-api/src/main/java/com/tomkeuper/bedwars/api/addon/AddonStorer.java

```diff
@@ -0,0 +1,28 @@
+package com.tomkeuper.bedwars.api.addon;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AddonStorer {
+    private static List<Addon> registeredAddons;
+    private static List<Addon> unloadedAddons;
+    private static List<Addon> loadedAddons;
+
+    public AddonStorer() {
+        if (registeredAddons == null) registeredAddons = new ArrayList<>();
+        if (unloadedAddons == null) unloadedAddons = new ArrayList<>();
+        if (loadedAddons == null) loadedAddons = new ArrayList<>();
+    }
+
+    public List<Addon> loadedAddons() {
```

Comment:
maybe rename these 3 to `getLoadedAddons(), getRegisteredAddons(), getUloadedAddons()` ?

Reply:
ah no, I saw that the 3 getters are already used elsewhere